csp-guru/datapackage.json

```diff
@@ -1,15 +1,167 @@
+                        "description": " how many hours the turbine can run on stored heat alone [h]"
+                    },
+                    {
+                        "name": "Field_size",
+                        "description": "the mirror area [㎡]"
+                    },
+                    {
+                        "name": "Cooling",
+                        "description": "cooling technology: dry, wet, or hybrid"
+                    },
+                    {
+                        "name": "Expected_generation",
+                        "description": "yearly power output [GWh/a]"
+                    },
+                    {
+                        "name": "Load_factor",
+                        "description": "percentage of maximum yearly power output, increases with storage"
+                    },
+                    {
+                        "name": "Total_cost",
+                        "description": "in thousand local currency"
+                    },
+                    {
+                        "name": "Specific_investment_cost_2015USD",
+                        "description": "investment cost per MW [2015 USD], discounted"
+                    },
+                    {
+                        "name": "Specific_investment_cost_2015EUR",
+                        "description": "investment cost per MW [2015 EURO], discounted"
+                    },
+                    {
+                        "name": "Developer",
+                        "description": "company that developed the project"
+                    },
+                    {
+                        "name": "HCE_manufacturer",
+                        "description": "heat collecting element (HCE) manufacturer"
+                    },
+                    {
+                        "name": "SCA_manufacturer",
+                        "description": "solar collector assemblies (SCA) manufacturer"
+                    },
+                    {
+                        "name": "Financed_through",
+                        "description": "how the owner makes money from the generated electricity: PPA, FiT, spot sales"
+                    },
+                    {
+                        "name": "Remuneration",
+                        "description": "indicates the level of FiT or PPA [ct/kWh], local currency"
+                    },
+                    {
+                        "name": "Remuneration_2015USD",
+                        "description": "indicates the level of FiT or PPA [ct/kWh], 2015 USD "
+                    },
+                    {
+                        "name": "Remuneration_2015EUR",
+                        "description": "indicates the level of FiT or PPA in [ct/kWh], 2015 EURO "
+                    },
+                    {
+                        "name": "LCOE_5_2015USD",
+                        "description": "levelized cost of electricity with 5 years payback [ct/kWh], 2015 USD discounted"
```

Comment:
"description": "levelized cost of electricity with 5% weighted average cost of capital [ct/kWh], 2015 USD discounted"

In the csp.guru wiki says wacc (weighted average cost of capital), I would use that or discount rate, or if you really want to write "payback" at least "discounted payback period". This change would apply to the other three below.

Reply:
thanks for pointing out that it's 5% not 5 years...